### DISCUSSION TITLE
Woodcutting plugin: Overlay current number of woodcutters

### DISCUSSION THREAD

**Discussion by Tea23** [runelite/runelite#16920] [Ideas]
The Forestry update gives benefits to players who are chopping the _same_ tree as other players, up to a +10 invisible level boost. Each player chopping a tree adds one more invisible level to every other player chopping the same tree (up to 10).

In tight spaces where one tile can be used to chop more than one tree, it can be difficult to tell _which_ tree is being chopped, as a stack of players could be chopping alternate trees (eg the Mahogany trees in Prifddinas).

If we could overlay a player count on trees it may make it easier to identify which is the ideal tree to chop. The overlay could max out at "10+" as player numbers above 10 do not give increased boosts.

**Comment by geheur** [comment]
https://runelite.net/plugin-hub/show/treecount
